### PR TITLE
Add workflow to add new PRs to project

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Adds a small workflow that adds new PRs automatically onto the documentation PRs project.
As this requires a PAT at this point, I've used an app instead (as an app can be scoped better and limited to a single repo). 

This app currently has access to this repo: projects boards (read/write) and meta/issues/prs (read). Which should be enough to do this task.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by frenck** [comment]
Hmm ok that didn't went as expected I guess

**Comment by frenck** [comment]
Now it works 👍

File: .github/workflows/add_prs_to_project.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          app_id: ${{ secrets.PROJECTS_APP_ID }}
+          private_key: ${{ secrets.PROJECTS_APP_PEM }}
+      - name: Add to Project
+        env:
+          TOKEN: ${{ steps.token.outputs.token }}
+        uses: actions/add-to-project@v0.3.0
+        with:
+          project-url: https://github.com/orgs/home-assistant/projects/10
+          github-token: ${{ env.TOKEN }}
```
Comment (ludeeus) [inline .github/workflows/add_prs_to_project.yml]:
```suggestion
          github-token: ${{ steps.token.outputs.token }}
```

File: .github/workflows/add_prs_to_project.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    steps:
+      - name: Generate app token
+        id: token
+        # Pinned to a specific version of the action for security reasons
+        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06 # v1.6.0
+        with:
+          app_id: ${{ secrets.PROJECTS_APP_ID }}
+          private_key: ${{ secrets.PROJECTS_APP_PEM }}
+      - name: Add to Project
+        env:
```
Comment (ludeeus) [inline .github/workflows/add_prs_to_project.yml]:
```suggestion
```

File: .github/workflows/add_prs_to_project.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+          TOKEN: ${{ steps.token.outputs.token }}
```
Comment (ludeeus) [inline .github/workflows/add_prs_to_project.yml]:
```suggestion
```

**Review by ludeeus — DISMISSED** [review]

**Review by ludeeus — APPROVED** [review]

**Pull request merged**